docs/FAQ/how-to-deploy-bold-bi-in-eks-using-application-load-balancer.md

```diff
@@ -0,0 +1,220 @@
+        |`BOLD_SERVICES_DB_USER`|Yes|Username for the database server<br /><br />Please refer to [this documentation](https://help.boldbi.com/embedded-bi/faq/what-are-         the-database-permissions-required-to-set-up-bold-bi-embedded/) for information on the user's permissions.|
+        |`BOLD_SERVICES_DB_PASSWORD`|Yes|The database user's password|
+        |`BOLD_SERVICES_DB_NAME`|No|If the database name is not specified, the system will create a new database called bold services.<br /><br />If you specify a             database name, it should already exist on the server.|
+        |`BOLD_SERVICES_POSTGRESQL_MAINTENANCE_DB`|No|For PostgreSQL DB Servers, this is an optional parameter.<br />The system will use the database name `postgres`           by default.<br />If your database server uses a different default database, please provide it here.|
+        |`BOLD_SERVICES_DB_ADDITIONAL_PARAMETERS`|No|If your database server requires additional connection string parameters, include them here.<br /><br />Connection         string parameters can be found in the official document.<br />My SQL: https://dev.mysql.com/doc/connector-net/en/connector-net-8-0-connection-options.html<br           />PostgreSQL: https://www.npgsql.org/doc/connection-string-parameters.html<br />MS SQL: https://docs.microsoft.com/en-                       us/dotnet/api/system.data.sqlclient.sqlconnection.connectionstring<br /><br /><b>Note:</b> A semicolon(;) should be used to separate multiple parameters.|
+        |`BOLD_SERVICES_USER_EMAIL`|Yes|It should be a valid email.|
+        |`BOLD_SERVICES_USER_PASSWORD`|Yes|It should meet our password requirements.|
+
+        <details>
+          <summary>
+            Example for above environment variables: 
+          </summary>
+
+          apiVersion: v1
+          kind: Secret
+          metadata:  
+            name: bold-license-key
+            namespace: bold-services
+          type: Opaque
+          stringData:
+            BOLD_SERVICES_UNLOCK_KEY: "@332e332e30fgfTa4NmxTdRataMFgre/GC5AyCj+BHVoCO4ax6M61s=eyJFbWFpbCI6InN1YmJpcmFtYW5peWFuLnRAc3luY2Z1c2lvbi5jb20iLCJQcm9kdWN0cyI6"
+          ---
+          apiVersion: v1
+          kind: Secret
+          metadata:  
+            name: bold-user-secret
+            namespace: bold-services
+          type: Opaque
+          stringData:
+            # It should be a valid email.
+            BOLD_SERVICES_USER_EMAIL: "admin@boldbi.com"
+
+            # It should meet our password requirements.
+            BOLD_SERVICES_USER_PASSWORD: "Admin@123"
+          ---
+          apiVersion: v1
+          kind: Secret
+          metadata:  
+            name: bold-db-secret
+            namespace: bold-services
+          type: Opaque
+          stringData:
+            # Type of database server can be used for configuring the Bold BI. Eg: mssql, mysql and postgresql
```

Comment:
# Type of a database server can be used for configuring the Bold BI. For example, mssql, mysql, and postgresql.

Reply:
concern addressed